std/assembly/typedarray.ts

```diff
@@ -98,4 +327,27 @@ export class Float64Array extends TypedArray<f64,f64> {
   subarray(begin: i32 = 0, end: i32 = 0x7fffffff): Float64Array {
     return changetype<Float64Array>(super.subarray(begin, end));
   }
+
+  /**
+   * @param callbackfn {function} - a function that reduces each value to a ReturnType
+   * @param initialValue {ReturnType} - the initial ReturnType value to be passed to the callbackfn
+   */
+  reduce<ReturnType>(
+    callbackfn: (accumulator: ReturnType, value: f64, index: i32, array: Float64Array) => ReturnType,
+    initialValue: ReturnType,
+    ): ReturnType {
+    return super.reduce_internal<Float64Array, ReturnType>(callbackfn, this, initialValue);
+  }
+
+  /**
+   * @param callbackfn {function} - a function that reduces each value to a ReturnType from the end
+   * of the array instead of from the beginning
+   * @param initialValue {ReturnType} - the initial ReturnType value to be passed to the callbackfn
+   */
```

Comment:
I think the comments here don't bring much to the table. The documentation a user sees for all the standard library implementaions is that in `std/assembly/index.d.ts` anyway and someone hacking on the source will most likely know what these functions are doing. Also note that the standard library sources are bundled with asc, and add to its size.

Reply:
Okay. I can remove the comments here.

Reply:
This should be completed.